components/ILIAS/Certificate/classes/Setup/Migration/CertificateIdMigration.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+
+namespace ILIAS\Certificate\Setup\Migration;
+
+use ilDatabaseException;
+use ilDatabaseUpdatedObjective;
+use ilDBConstants;
+use ilDBPdoInterface;
+use ilDBStatement;
+use ILIAS\Setup\Environment;
+use ILIAS\Setup\Migration;
+use JsonException;
+use ReflectionClass;
+use ILIAS\Data\UUID\Factory;
+
+
+class CertificateIdMigration implements Migration
+{
+    public const NUMBER_OF_STEPS = 10000;
```

Comment:
Do we still need this? Why not returning `NUMBER_OF_CERTS_PER_STEP` in `getDefaultAmountOfStepsPerRun`?